.pre-commit-config.yaml

```diff
@@ -43,16 +43,16 @@ repos:
       - id: isort
         additional_dependencies: ["toml"]
   - repo: https://github.com/PyCQA/flake8
-    rev: 5.0.4
+    rev: 6.0.0
     hooks:
       - id: flake8
         additional_dependencies: [
-            'flake8-bugbear==21.11.29',
+            'flake8-bugbear==22.10.27',
             'flake8-coding==1.3.2',
-            'flake8-comprehensions==3.10.0',
-            'flake8-deprecated==1.3',
-            'flake8-print==4.0.0',
-            'flake8-quotes==3.3.1',
```

Comment:
this is not currently compatible with flake8 6.0.0

https://github.com/zheller/flake8-quotes/issues/110